src/admin/sql/updates/mysql/1.0.2.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `#__tj_activities` MODIFY `template` text;
+ALTER TABLE `#__tj_activities` MODIFY `formatted_text` text;
+
+ALTER TABLE `#__tj_activities` ENGINE = InnoDB;
+ALTER TABLE `#__tj_activities` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```

Comment:
@ankush-maherwal do it on column level

Reply:
@manojLondhe Removed code to update charset as this needs more testing